workers/loc.api/sync/currency.converter/index.js

```diff
@@ -165,6 +182,7 @@ class CurrencyConverter {
     }
 
     for (const [symbol, conversion] of synonymous) {
+      await setImmediatePromise()
```

Comment:
why add setImmediatePromise if next line 
 
> const price = await finderFn(symbol)


Would already add this to the event loop.